bpo-46491: Allow Annotated on outside of Final/ClassVar (#30864)
python/cpython: Misc/NEWS.d/next/Library/2022-01-24-23-55-30.bpo-46491.jmIKHo.rst

> Allow Annotated on outside of Final/ClassVar.

Reviewer: This should make clear that it's about typing. You can use RST like in the docs, e.g. ```:data:`typing.Annotated`.```

Author: Thanks, this should be okay now

> Allow :data:`typing.Annotated` on outside of :data:`typing.Final`/:data:`typing.ClassVar`.

Reviewer: ```suggestion
Allow :data:`typing.Annotated` to wrap :data:`typing.Final` and :data:`typing.ClassVar`.
```
Slight wording improvements. Also feel free to add "Patch by Gregory Beauregard" if you like.

Author: Yeah that sounds better, thanks!
